linuxbrew-build.sh

```diff
@@ -118,27 +111,19 @@ readonly LINUXBREW_PACKAGES=(
   s3cmd
 )
 
-# Temporarily excluded packages.
-readonly LINUXBREW_PACKAGES_DISABLED=(
-  maven
-)
-
 successful_packages=()
 failed_packages=()
 
 for package in "${LINUXBREW_PACKAGES[@]}"; do
-
-  if ( set -x; ./bin/brew install --build-from-source "$package" ); then
```

Comment:
Why we don't need `--build-from-source` anymore?

Reply:
Added back.